librpc/test/client_read_ops_test.h

```diff
@@ -85,6 +88,19 @@ class ClientReadOpsTest : public testing::Test {
     return reinterpret_cast<void *>(&r);
   }
 
+  static std::string make_json_record(int64_t ts, const std::string &msg) {
+    namespace pt = boost::property_tree;
+    pt::ptree root;
+
+    root.put("TIMESTAMP", ts);
+    root.put("MSG", msg);
+
+    std::stringstream ss;
+    pt::write_json(ss, root);
+    std::string ret = ss.str();
```

Comment:
Same thing here -- need to figure out what correct JSON string format is. We need to document it somewhere too.